app/tests/mock_database.py

```diff
@@ -754,3 +763,406 @@ def create_multiple_test_records():
         file_10,
         file_11,
     ]
+
+
+def create_multiple_files_for_consignment(consignment_id):
```

Comment:
can we use Factory boy instead?

Reply:
It is too late in the current sprint to implement, but will refactor with a new ticket in future